### PR TITLE
Add admin mentor page

### PR DESCRIPTION
**Pull request by loriking**
Part 1 (issue#28) of issue#33 Build out admin section for mentors

This concerns the creation of a section for administrators of Exercism to get an overview of mentors on the site. 
see: https://github.com/exercism/guest-contributors/issues/28
see also: https://github.com/exercism/guest-contributors/issues/33

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -34,11 +34,12 @@
     resources :solutions, only: [:show] do
       resources :iterations, only: [:show]
     end
+    resources :mentors, only: [:index]
   end
 
   # ###### #
   # Mentor #
-  # ###### #
+  # ###### #,
```
Comment (iHiD) [inline config/routes.rb]: This is breaking the build.

File: app/assets/javascripts/admin/mentors.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (iHiD) [inline app/assets/javascripts/admin/mentors.coffee]: Can you delete this empty file pls :)

File: app/assets/stylesheets/admin/mentors.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the admin/mentors controller here.
```
Comment (iHiD) [inline app/assets/stylesheets/admin/mentors.scss]: Can you delete this one too.

File: app/helpers/admin/mentors_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module Admin::MentorsHelper
```
Comment (iHiD) [inline app/helpers/admin/mentors_helper.rb]: And this one.

Comment (loriking) [reply app/helpers/admin/mentors_helper.rb]: Removed

File: app/assets/stylesheets/admin/mentors.scss (hunk 3 above)
Comment (loriking) [reply app/assets/stylesheets/admin/mentors.scss]: Removed

File: app/assets/javascripts/admin/mentors.coffee (hunk 2 above)
Comment (loriking) [reply app/assets/javascripts/admin/mentors.coffee]: Removed

File: config/routes.rb (hunk 1 above)
Comment (loriking) [reply config/routes.rb]: Fixed. Ran the system test file and it appears to be working. I have pushed the changes.

**Comment by iHiD** [comment]
@loriking Can you remove the `.DS_Store` files too pls? Your coaches should be able to explain how to add these to your `.gitignore` :)

**Comment by iHiD** [comment]
@loriking You'll want to add this to your local gitignore file rather than changing the one in the project as DS_Store files are local to your setup. Instructions are [here](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

**Comment by loriking** [comment]
@iHiD Thank you for the clarification and instructions. It's fixed now.

**Comment by iHiD** [comment]
Thanks. Can you remove the commit from this branch pls? You'll want to use an interactive rebase and a force push :)

**Comment by iHiD** [comment]
Great. Thank you!

**Pull request merged**